src/utils/constants.js

```diff
@@ -0,0 +1,10 @@
+const Variant = {
```

Comment:
Should this have [`Object.freeze`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/freeze) applied to it?

Reply:
Possibly, this is one of those things that I'm not sure of the expectations / necessity of that but would be willing to discuss more about it :)

Reply:
Actually the more I read about `Object.freeze` I'm leaning more towards just a yes on this one